src/components/Register/Register.js

```diff
@@ -26,7 +27,7 @@ if (showSuccess) {
 
 if (!showSuccess) {
     return (
-        <> 
+        <PopupWithForm closeModal={closeModal} isModalOpen={isModalOpen} setShowSuccess={setShowSuccess}> 
         <h3 className="popup__title">Sign up</h3>
         <form onSubmit={handleRegistration} className="popup__form">
             <label className="popup__label" htmlFor="email">Email</label>
```

Comment:
**Might be better**


you can create a special component `Input` for any inputs not to duplicate `label, input and p` everywhere


```jsx

 const Input = ({ 
    label='', 
    required=true, // you can set defaul values everywhere not to duplicate them in each input
    className = "popup__input", // you can set defaul values everywhere not to duplicate them in each input
    type = "text", // you can set defaul values everywhere not to duplicate them in each input
    name,
    id,
    placeholder = "",
    onChange,
    value = "",
    error = "",
  }) => {
    return (
      <>
        <label className="popup__label" htmlFor={id}>
          {label}
        </label>
        <input
          required={required}
          className={className}
          type={type}
          name={name}
          id={id}
          placeholder={placeholder}
          onChange={onChange}
          value={value}
        />
        <p className="popup__input-error">{error}</p>
      </>
    );
  };


```


and you can use it everywhere


```jsx

            <Input
                label="Email"
                id="email"
                name="email"
                type="email"
                placeholder="Enter email"
                onChange={handleChange}
                value={values.email}
                error={errors.email}
            />

```